src/coreclr/jit/gcencode.cpp

```diff
@@ -3593,7 +3603,12 @@ size_t GCInfo::gcDumpPtrTable(const BYTE* table, const InfoHdr& header, unsigned
 void GCInfo::gcFindPtrsInFrame(const void* infoBlock, const void* codeBlock, unsigned offs)
 {
     GCDump gcDump(GCINFO_VERSION);
+
+#ifdef DEBUG
     gcDump.gcPrintf = gcDump_logf; // use my printf (which logs to VM)
+#else
+    gcDump.gcPrintf = printf;
```

Comment:
Are you using jitstdout for release because of speed?

Reply:
Enabling gcDump_logf to build when DEBUG is not defined but DUMP_GC_TABLES is defined seemed like more work, and possibly not necessary to achieve the goal, so I went for a simpler build fix.